Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1129,7 +1129,20 @@ - (void)_didCommitLayerTree:(const WebKit::RemoteLayerTreeTransaction&)layerTree
 
         if (isFirstTransactionAfterObscuredInsetChange) {
             _perProcessState.firstTransactionIDAfterObscuredInsetChange = std::nullopt;
-            if (_overriddenLayoutParameters && !self._shouldDeferGeometryUpdates) {
+            bool needsOverrideLayoutSizeUpdate = [&] {
+                if (!_overriddenLayoutParameters)
+                    return false;
+
+                if (!self._shouldDeferGeometryUpdates)
+                    return true;
+
+                if (WTF::IOSApplication::isSafariViewService() && !self.window.windowScene.effectiveGeometry.isInteractivelyResizing)
```

Comment:
(Might be good to leave a `FIXME` or just a comment here briefly explaining the reason for this bundle check)

Reply:
Yes that makes sense, I'll leave a comment.